pkgs/development/ocaml-modules/bistro/default.nix

```diff
@@ -12,6 +12,11 @@ buildDunePackage rec {
     sha256 = "114gq48cpj2mvycypa9lfyqqb26wa2gkdfwkcqhnx7m6sdwv9a38";
   };
 
+  patches = [(fetchpatch {
+    url = https://github.com/pveber/bistro/commit/0931db43a146ad7829dff5120161a775f732a878.patch;
```

Comment:
Quote `url`?  Also, consider including a comment regarding this patch?

Reply:
Done. Thanks.